rd_ui/app/views/dashboard.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
                 </div>
             </div>
 
-            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
```

Comment:
I´m sure, I used ng-hide. Maybe I had a mistake. But I'm sure that I used ng-hide.